src/app/redux/actionCreators.js

```diff
@@ -650,7 +650,7 @@ export const addEmptyRowAndOpenEntityView = (
   onSuccess
 ) => async dispatch => {
   await dispatch(loadColumns(tableId));
-  const freshRow = await createAndLoadRow(dispatch, tableId);
+  const [freshRow] = await createAndLoadRow(dispatch, tableId);
```

Comment:
Ist das korrekt hier? Damit ist es ja immer ein Array.
Weiter unten in Zeile 659 oder 662 z.B. wird aber auf `freshRow.id` zugegriffen.
Das sollte ja eigentlich nicht gehen, wenn freshRow ein Array ist.

Reply:
Hier wird ja das Array destrukturiert und der erste (und in diesem Falle einzige) Wert als Variable freshRow rausgeholt.